RENotebookApi/RENotebookApi.fsproj

```diff
@@ -1,15 +1,22 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net472</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net472</TargetFrameworks>
```

Comment:
`netstandard2.0` is probably best avoided in the ifsharp context as only `net472` is really released